lib/ur/usage_report.go

```diff
@@ -328,6 +328,9 @@ func (s *Service) reportData(ctx context.Context, urVersion int, preview bool) (
 }
 
 func (*Service) UptimeS() int {
+	if StartTime.Year() < 2023 {
```

Comment:
```suggestion
	// Handle nonexistent or wildly incorrect system clock. This code was written in 2023, it can't run in the past.
	if StartTime.Year() < 2023 {
```